core/src/com/mygdx/game/client/initialize/StartUnitInitializer.java

```diff
@@ -0,0 +1,19 @@
+package com.mygdx.game.client.initialize;
+
+import com.badlogic.ashley.core.Entity;
+import com.mygdx.game.assets.GameScreenAssets;
+import com.mygdx.game.client.component.FieldComponent;
+import com.mygdx.game.client.entityfactory.UnitFactory;
+import com.mygdx.game.client.model.Coordinates;
+import com.mygdx.game.config.UnitConfig;
+
+import java.util.Map;
+
+public class StartUnitInitializer {
```

Comment:
To rozumiem jako coś służącego do testowania zachowania w trakcie implementacji.

Reply:
No tak. Na razie tutaj na sztywno tworzę sobie jednostkę.